07/index.html

```diff
@@ -0,0 +1,136 @@
+      </p>
+      <p>
+        Suzuki es un fabricante independiente, aunque durante unos años tuvo vínculos cercanos con General Motors y Volkswagen, sin llegar a ser controlado por otro fabricante.3
+      </p>
+    </nav>
+    <section>
+      <img src="./assets/example/GIXXERMANIA-ABRIL2024.jpeg">
+      <img src="./assets/example/TEST-DRIVE_Guadalajara.jpeg">
+    </section>
+    <section>
+      <fieldset>
+        <legend id="Gmodelos">Garage</legend>
+        <fieldset>
+          <legend>Cuidad</legend>
+          <label for="gsx">Gsx 8s</label>
+          <input type="radio" id="gsx" name="Cb" value="gsx-8s">
+          <h4>214,990 mxn</h4>
+          <img src="./assets/example/gsx-8s.png" width="200">
+          <br>
+          <label for="intruder">Intruder</label>
+          <input type="radio" id="intruder" name="Cb" value="Intruder">
+          <h4>58,990 mxn</h4>
+          <img src="./assets/example/intruder.png" width="200">
+          <br>
+          <label for="gixxer">Gixxer 150</label>
+          <input type="radio" id="gixxer" name="Cb" value="gixxer">
+          <h4>54,990 mxn</h4>
+          <img src="./assets/example/gixxer-150.png" width="200">
+        </fieldset>
+        <fieldset>
+          <legend>Doble Propósito</legend>
+          <label for="V-Strom800">V-Strom 800DE</label>
+          <input type="radio" id="V-Strom800" name="Dp" value="v-strom-800">
+          <h4>254,990 mxn</h4>
+          <img src="./assets/example/v-strom-800de.png" width="200">
+          <br>
+          <label for="dr650">DR 650</label>
+          <input type="radio" id="dr650" name="Dp" value="dr650">
+          <h4>169,990 mxn</h4>
+          <img src="./assets/example/dr650.png" width="200">
+          <br>
+          <label for="250sx">V-Strom 250SX</label>
+          <input type="radio" id="250sx" name="Dp" value="250sx">
+          <h4>99,990 mxn</h4>
+          <img src="./assets/example/v-strom-250-sx.png" width="200">
+        </fieldset>
+        <fieldset>
+          <legend>Scooter</legend>
+          <label for="Avenis">Avenis 125</label>
+          <input type="radio" id="Avenis" name="scooter" value="avenis">
+          <h4>49,990 mxn</h4>
+          <img src="./assets/example/avenis-125.png" width="200">
+          <br>
+          <label for="burgman">Brugman street 125</label>
+          <input type="radio" id="burgman" name="scooter" value="burgman">
+          <h4>44,990 mxn</h4>
+          <img src="./assets/example/burgman-street-125.png" width="200">
+        </fieldset>
+      </fieldset>
+    </section>
+    <article>
+      <fieldset>
+        <legend id="servicio">Servicios</legend>
+        <label for="Snombre">Nombre y apellido</label>
+        <input type="text" id="Snombre">
+        <br>
+        <label for="E-mail">E-mail</label>
+        <input type="email" id="E-mail">
+        <br>
+        <label for="telefono">Numero telefonico</label>
+        <input type="tel" id="telefono">
```

Comment:
Perhaps you should add some help to the user
---
Tal vez deberías agregar alguna ayuda al usuario